sssom/writers.py

```diff
@@ -107,7 +107,7 @@ def write_json(msdf: MappingSetDataFrame, filename: str, serialisation="json") -
             json.dump(data, outfile, indent="  ")
 
     else:
-        raise Exception(
```

Comment:
never ever throw Exception in your code - always pick one of its subclasses, otherwise you can't reliably handle it later